env_kind.go

```diff
@@ -768,7 +767,7 @@ func (r *kindRunnable) prePullImage(ctx context.Context) (err error) {
 	}
 
 	if _, err = r.env.execContext(ctx, "docker", "image", "inspect", r.opts.Image).CombinedOutput(); err == nil {
-		return nil
+		return r.loadImageIntoKindCluster(ctx)
 	}
```

Comment:
Actually this won't work for a public image that is not present on the machine, yet. 

I feel like we should remove the `if` block. The comment in line 774/773 suggests that we should pull the image if we get an "No such image" error, but pulling an image that does no have a manifest will never work.

So rather we should either
a. check if the image is present locally and if not try to pull it and finally (always) load it into docker
b. check if image is present locally. If yes do nothing more. If no check if it is present locally and load it into kind or fail.

Reply:
done